### PR TITLE
Unify help display logic for various "complex" rpc commands

### PR DESCRIPTION
**Pull request by UdjinM6**
There is a bit of inconsistency in the behaviour of various rpcs with subcommands, this should fix it.

### CODE REVIEW

File: src/rpc/governance.cpp [hunk 1]
```diff
@@ -1026,9 +1020,10 @@ UniValue gobject(const JSONRPCRequest& request)
     } else if (strCommand == "getcurrentvotes") {
         // GETVOTES FOR SPECIFIC GOVERNANCE OBJECT
         return gobject_getcurrentvotes(request);
+    } else {
+        gobject_help();
+        return UniValue::VNULL; // avoid compiler warnings
```
Comment (codablock) [inline src/rpc/governance.cpp]: aren't there some `noreturn` attributes in C++ for exactly this? Like `[[ noreturn ]] void myfunc()`? See https://en.cppreference.com/w/cpp/language/attributes/noreturn

Comment (UdjinM6) [reply src/rpc/governance.cpp]: Not sure I understand... `gobject` does return smth in all other cases, it's only when command doesn't match any of known ones we have this case.

Comment (codablock) [reply src/rpc/governance.cpp]: I mean that `gobject_help` and friends could have that attribute, making it unnecessary for these dummy returns after calling it.

Comment (UdjinM6) [reply src/rpc/governance.cpp]: Ahh, I see. Good point 👍 Fixed.

**Pull request merged**